matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -214,21 +223,25 @@ async fn message_loop<M: Messenger>(
                             .get_mut(&peer)
                             .expect("couldn't find data channel for peer")
                             .get_mut(channel_index).unwrap_or_else(|| panic!("couldn't find data channel with index {channel_index}"));
-                        data_channel.send(packet).unwrap();
-
+                        if let Err(e) = data_channel.send(packet) {
+                            // Peer we're sending to closed their end of the connection.
+                            // We anticipate the PeerLeft event soon, but we sent a message before it came.
+                            // Do nothing. Only log it.
+                            warn!("failed to send to peer {peer} (socket closed): {e:?}");
+                        };
```

Comment:
This feels like part of the expected messaging loop life-cycle. Should we log it as `info` instead of `warn`?

Reply:
Feels more like a warn to me since the socket wasn't ended cleanly.